### PR TITLE
[release-9.1] Remove non-functional search link from hierarchy search limit message.

### PR DESCRIPTION
**Pull request by EreMaijala**

### CODE REVIEW

File: languages/hy.ini [hunk 1]
```diff
@@ -1382,7 +1382,7 @@ total_saved_items = "Ընդհանուր պահված նյութեր"
 total_tags = "Ընդհանուր պիտակներ"
 total_users = "Ընդհանուր օգտվողներ"
 Transliterated Title = "Տառադարձված վերնագիր"
-tree_search_limit_reached_html = 'Ձեր որոնումը բերեց չափազանց շատ արդյունքներ՝ ծառում ցուցադրելու համար: Ցուցադրվում է միայն առաջին <b>%%limit%%</b> նյութերը։ Ամբողջական որոնման համար սեղմեք <a id="fullSearchLink" href="%%url%%" target="_blank">այստեղ։</a>'
+tree_search_limit_reached_html = 'Ձեր որոնումը բերեց չափազանց շատ արդյունքներ՝ ծառում ցուցադրելու համար: Ցուցադրվում է միայն առաջին <b>%%limit%%</b> նյութերը։ Ամբողջական որոնման համար սեղմեք <a id="fullSearchLink" href="%%url%%" target="_blank" data-lightbox-ignore>այստեղ։</a>'
```
Comment (demiankatz) [inline languages/hy.ini]: This looks like the wrong change...

File: languages/mn.ini [hunk 2]
```diff
@@ -1380,7 +1380,7 @@ total_saved_items = "Нийт хадгалсан зүйлс"
 total_tags = "Нийт шошго"
 total_users = "Нийт хэрэглэгч"
 Transliterated Title = "Хөрвүүлсэн гарчиг"
-tree_search_limit_reached_html = 'Таны хайлт модоор харуулж болох хэт олон үр дүн буцаан гаргасан. Эхний <b>%%limit%%</b> зүйлүүдийг зөвхөн харуулж байна. Бүрэн хайлт хийхийн тулд <a id="fullSearchLink" href="%%url%%" target="_blank">энд.</a> дарна уу'
+tree_search_limit_reached_html = 'Таны хайлт модоор харуулж болох хэт олон үр дүн буцаан гаргасан. Эхний <b>%%limit%%</b> зүйлүүдийг зөвхөн харуулж байна. Бүрэн хайлт хийхийн тулд <a id="fullSearchLink" href="%%url%%" target="_blank" data-lightbox-ignore>энд.</a> дарна уу'
```
Comment (demiankatz) [inline languages/mn.ini]: Same here.

**Review by demiankatz — CHANGES_REQUESTED** [review]
Looks like a couple of the language files may have gotten the wrong versions committed. There was also a build failure caused by language normalization, which I went ahead and fixed. I don't mind taking care of fixing those last couple of language files too if you want me to, but for now I'll pass this back to you so I can knock a couple of other items off my list.

**Comment by EreMaijala** [comment]
@demiankatz Oops. Should be fixed now.

**Review by demiankatz — APPROVED** [review]
Thanks, @EreMaijala!

**Pull request merged**